src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+    List<Book> findByTitle(String title);
```

Comment:
About this method, we need a single book so a List<Book> wouldn't do the trick... we could maybe try with Optional<Book> so it will find just one book ( if it does exists )